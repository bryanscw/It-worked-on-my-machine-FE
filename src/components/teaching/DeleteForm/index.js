import React from 'react'
import PropTypes from 'prop-types'
/** This component displays a delete form */
export default function DeleteForm(props) {
    const {
        onSubmit
    } = props;

    return (
        <>
            <button className="btn btn-danger mr-2" onClick={() => onSubmit(true)}>Confirm</button>
            <button className="btn btn-secondary" onClick={() => onSubmit(false)}>Cancel</button>
        </>
    )
}

DeleteForm.propTypes = {
    /** The function to call when the form is submitted */
    onSubmit: PropTypes.func.isRequired,
}