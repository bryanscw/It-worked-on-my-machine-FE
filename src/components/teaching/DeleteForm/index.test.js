import React from 'react'
import { render, cleanup } from '@testing-library/react'
import DeleteForm from './index.js';

afterEach(cleanup);

it('should take a snapshot', () => {
    const { asFragment } = render(<DeleteForm />);
    expect(asFragment()).toMatchSnapshot();
})