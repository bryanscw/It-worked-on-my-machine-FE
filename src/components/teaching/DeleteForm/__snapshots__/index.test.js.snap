// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <button
    class="btn btn-danger mr-2"
  >
    Confirm
  </button>
  <button
    class="btn btn-secondary"
  >
    Cancel
  </button>
</DocumentFragment>
`;
